test/integration/enterprise_test.go

```diff
@@ -25,6 +25,7 @@ import (
 )
 
 func TestKongEnterprisePostgres(t *testing.T) {
+	SkipEnterpriseTestIfNoEnv(t)
```

Comment:
Can we rename this to explicitly say that we're looking for a license not just "an env"?